[css-text] Added 1 overflowing hanging spaces at end of line test

TalbotG opened the pull request:
/css/css-text/white-space/white-space-pre-wrap-trailing-spaces-021.html  
/css/css-text/white-space/reference/white-space-pre-wrap-trailing-spaces-021-ref.html

This spun from [Issue 4297](https://github.com/w3c/csswg-drafts/issues/4297)

Basically, overflowing hanging white spaces at end of line of block with 'white-space: pre-wrap' should be paintable and should act like ink overflow.

At my website, the files are:

http://www.gtalbot.org/BrowserBugsSection/CSS3Text/white-space-pre-wrap-trailing-spaces-021.html

http://www.gtalbot.org/BrowserBugsSection/CSS3Text/reference/white-space-pre-wrap-trailing-spaces-021-ref.html

TalbotG committed:
Added 1 overflowing hanging spaces at end of line test

fantasai reviewed (changes requested):
Hmm. So, css-text-3 actually allows the spaces to be truncated; thus Firefox and WebKit's renderings are correct. The interesting thing about ink overflow is that it does not expand the scrollable overflow area, and that's what we need to focus on testing.

To do this, it's easy: give the first div `overflow: auto` and the second one `overflow: hidden`. They should render identically because there should be no scrollbars caused by the hanging spaces.

TalbotG committed:
Adjusted white-space-pre-wrap-trailing-spaces-021 test and its reference according to review comments

TalbotG commented:
In 4c3958e , I made several small text adjustments to the test:

> the second one overflow: hidden

I instead removed 4 `&nbsp;` instead ... which must be equivalent to `overflow: hidden` for the `div#reference`

I added a [help link to ink overflow spec definition](https://www.w3.org/TR/css-overflow-3/#ink) 

The no horizontal scrollbar is now formally a pass condition of the test.

fantasai committed:
Grammar fixes

And also add 'overflow: hidden' to be sure to match the test, in case the spaces are not exactly 1ch wide.

fantasai committed:
Grammar fixes

fantasai committed:
Grammar fix fix

fantasai reviewed (approved):
Made a few minor fixes, but otherwise looks OK.
